nss/integration-tests/nss_test.go

```diff
@@ -100,21 +103,17 @@ func TestIntegration(t *testing.T) {
 			}
 
 			got, err := outNSSCommandForLib(t, socketPath, originOuts[tc.db], cmds...)
-
-			// Some tests use corrupted databases to trigger specific errors. When opening a corrupted database,
-			// the cache will schedule a clear-up routine to fix the database, so we need to give it some time to
-			// finish before proceeding with the test, otherwise the test cleanup functions would run (removing
-			// the files and directories) before the routine has finished and it will panic.
-			if tc.cacheDB != "multiple_users_and_groups" {
-				time.Sleep(time.Second)
-			}
-
 			if tc.wantErr {
 				require.Error(t, err, "Expected an error, but got none")
 				return
 			}
 			require.NoError(t, err, "Expected no error, but got %v", err)
 
+			if tc.wantSecondCall {
+				got, err = outNSSCommandForLib(t, socketPath, originOuts[tc.db], cmds...)
+				require.NoError(t, err, "Expected no error, but got %v", err)
```

Comment:
I think we want to assert empty/not empty got too.

Reply:
We already have the golden files for that, no? But I agree with the assertion before issuing the command again.

Reply:
I think the golden file should assert the first call, and then, we assert manually the second one